cmd/kogito/command/util/build_from_file.go

```diff
@@ -0,0 +1,100 @@
+// Copyright 2020 Red Hat, Inc. and/or its affiliates
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package util
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/context"
+	"github.com/kiegroup/kogito-cloud-operator/cmd/kogito/command/message"
+	"io"
+	"os"
+	"path/filepath"
+	"strings"
+)
+
+var supportedExtensions = []string{"dmn", "drl", "bpmn", "bpmn2", "properties"}
```

Comment:
It may be good to check suffixes starting with dot, so the `IsSuffixSupported` function won't match any name ending with these strings (for example something like `https://github.com/<username>/my-first-bpmn`)

Reply:
adjusted.